plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx

```diff
@@ -32,12 +32,17 @@ import { IconComponent } from '@backstage/core-plugin-api';
 import { useEntityPermission } from '@backstage/plugin-catalog-react';
 import { catalogEntityDeletePermission } from '@backstage/plugin-catalog-common';
 
-// TODO(freben): It should probably instead be the case that Header sets the theme text color to white inside itself unconditionally instead
-const useStyles = makeStyles({
-  button: {
-    color: 'white',
-  },
-});
+/** @public */
+export type EntityContextMenuClassKey = 'button';
+
+const useStyles = makeStyles(
+  () => ({
+    button: {
+      color: 'white',
```

Comment:
I was wondering about that as well, but I made the change here because we are in a plugin (not in a component in the main package).

I did not want to introduce a "plugin-specific" property in the general theme, and I am not sure that it is possible to pick an existing theme property (because it really depends on what theme designers are doing). Last but not least, I discovered that there was already an `overridableComponents.ts` file in the plugin, so I had the impression that this was the pattern to follow.

Reply:
Actually, I was looking for a property at the header level (in `layout/Header/Header.tsx`), but the color is defined in the `title` and `subtitle` properties:

```
const useStyles = makeStyles<BackstageTheme>(
  theme => ({
    header: {
      gridArea: 'pageHeader',
      padding: theme.spacing(3),
      width: '100%',
      boxShadow: theme.shadows[4],
      position: 'relative',
      zIndex: 100,
      display: 'flex',
      flexDirection: 'row',
      alignItems: 'center',
      backgroundImage: theme.page.backgroundImage,
      backgroundPosition: 'center',
      backgroundSize: 'cover',
      [theme.breakpoints.down('sm')]: {
        flexWrap: 'wrap',
      },
    },
    leftItemsBox: {
      maxWidth: '100%',
      flexGrow: 1,
    },
    rightItemsBox: {
      width: 'auto',
    },
    title: {
      color: theme.palette.bursts.fontColor,
      wordBreak: 'break-word',
      fontSize: theme.typography.h3.fontSize,
      marginBottom: 0,
    },
    subtitle: {
      color: theme.palette.bursts.fontColor,
      opacity: 0.8,
      display: 'inline-block', // prevents margin collapse of adjacent siblings
      marginTop: theme.spacing(1),
      maxWidth: '75ch',
    },
    type: {
      textTransform: 'uppercase',
      fontSize: 11,
      opacity: 0.8,
      marginBottom: theme.spacing(1),
      color: theme.palette.bursts.fontColor,
    },
    breadcrumb: {
      color: theme.palette.bursts.fontColor,
    },
    breadcrumbType: {
      fontSize: 'inherit',
      opacity: 0.7,
      marginRight: -theme.spacing(0.3),
      marginBottom: theme.spacing(0.3),
    },
    breadcrumbTitle: {
      fontSize: 'inherit',
      marginLeft: -theme.spacing(0.3),
      marginBottom: theme.spacing(0.3),
    },
  }),
  { name: 'BackstageHeader' },
);
```

So, let's say that I want to use the overridable property `BackstageHeader.title.color` in the `EntityContextMenu.tsx` file, what should I do?

I actually started with something like:

```
const useStyles = makeStyles(
  (theme) => { debugger; return ({
    button: {
      color: theme.overrides.BackstageHeader.title,
    },
  }) },
  { name: 'PluginCatalogEntityContextMenu' },
);
```

But that didn't seem right. If I override BackstageHeader in my theme, then it actually works. But the overrides.BackstageHeader property may not be defined (and the type system does not allow me to do that anyways).

What's the right way to reference a property of the Backstage Header title in my component?

Reply:
But so, is there a way for me to say "the color of the context menu icon should be the save as the color of the header title" (knowing that this can be overriden)?